src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Plane.PlaneBuilder planeBuilder = new Plane.PlaneBuilder();
+        Plane myPlane = planeBuilder.setId(1)
+                .setModel("Бройлер 747")
```

Comment:
Please use English in your code. It would not be very cool if you will read the comment on Hindi or German left by someone before in your production code.

Reply:
At least write 
```suggestion
                .setModel("Broiler 747")
```
:)

Reply:
ok) Done